java/src/main/asciidoc/index.adoc

```diff
@@ -0,0 +1,156 @@
+= Vert.x EventBus Client
+
+Vert.x EventBus Client is a Java client allowing applications to interact with a Vert.x EventBus bridge
+via either TCP or WebSocket transport. After connection, it allows:
+
+* Sending messages to an address of the EventBus.
+* Sending messages to an address of the EventBus and expect a reply.
+* Publishing messages to an address of the EventBus.
+* Creating a consumer and register it against an address.
+* Unregistering a consumer from its address.
+
+Under the hood, the packets sent to server follow the protocol defined in link:https://vertx.io/docs/vertx-tcp-eventbus-bridge/java/[Vert.x EventBus TCP bridge].
+
+NOTE: This client does not need Vert.x to work.
```

Comment:
This client does not rely on Vert.x and requires a Java 6 runtime, making it embeddable in Android applications.

Reply:
Done